src/app/styles/components/_preview-card.scss

```diff
@@ -55,6 +56,7 @@
         flex-flow: column nowrap;
         align-items: stretch;
         justify-content: space-between;
+        margin-bottom: 5rem;
```

Comment:
Since the More link is absolutely positioned instead of inline, I can't think of any way to coax the browser into calculating the layout we want. But we can do it in SASS using a variable.

Looks like the height of `.goto-more-posts` is currently `line-height: 2.75` ✖️ `font-size: 1.4rem` = 3.85rem. We could hardcode a `$goto-more-posts-btn-height` variable to 3.85rem (or round to 4 for some slack), then set the card's `margin-bottom` to, say, `$goto-more-posts-btn-height + 2*$home-section-padding`